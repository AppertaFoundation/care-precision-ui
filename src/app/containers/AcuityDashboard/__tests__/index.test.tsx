import * as React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';
import { AcuityDashboard } from '../index';

const shallowRenderer = createRenderer();

describe('<Acuity Dashboard />', () => {
  it('should match snapshot', () => {
    shallowRenderer.render(<AcuityDashboard />);
    const renderedOutput = shallowRenderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
