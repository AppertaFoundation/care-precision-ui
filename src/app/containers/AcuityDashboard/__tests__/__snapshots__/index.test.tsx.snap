// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Acuity Dashboard /> should match snapshot 1`] = `
<React.Fragment>
  <Helmet
    defer={true}
    encodeSpecialCharacters={true}
  >
    <title>
      Acuity Dashboard
    </title>
    <meta
      content="A Acuity Dashboard"
      name="description"
    />
  </Helmet>
  <AcuityList />
</React.Fragment>
`;
